src/client/helpers/entity.js

```diff
@@ -225,3 +226,59 @@ export function getISBNOfEdition(entity) {
 export function getEditionFormat(entity) {
 	return (entity.editionFormat && entity.editionFormat.label) || '?';
 }
+
+// relationshipTypeId = 10 refers the relation (<Work> is contained by <Edition>)
+const relationshipTypeId = 10;
+
+/**
+ * Filter the relationship according to relationshipTypeId
+ *
+ * @param {object} entity - Entity with all relationships
+ * @returns {object} retrun the all relationshiops after removing the relatioship with relationshipTypeId = 10
+ */
+export function getFilteredRelationship(entity) {
+	return entity.relationships.filter((relation) => relation.typeId !== relationshipTypeId);
+}
+
+
+/**
+ * Get all works constained by entity according to relationshipTypeId
+ *
+ * @param {object} entity - Edition with all relationships
+ * @returns {object} Return all the works of Edition with relationshipTypeId = 10
+ */
+export function getWorksContainedByEdition(entity) {
+	let works = null;
+	if (entity.relationships) {
```

Comment:
Similar to the above comment, I'd checking for `Array.isArray(entity.relationships)` here and in getEditionsContainsWork